auth4genai/snippets/get-started/vercel-ai-next-js/call-others-api.mdx

```diff
@@ -94,13 +95,10 @@ AUTH0_SECRET='random 32 byte value'
 AUTH0_DOMAIN='<your-auth0-domain>'
 AUTH0_CLIENT_ID='<your-auth0-application-client-id>'
 AUTH0_CLIENT_SECRET='<your-auth0-application-client-secret>'
+AUTH0_SCOPE='openid profile email offline_access'
```

Comment:
Not needed for nextjs-auth0, because the default requested scopes include `offline_access`.
```suggestion
```